code/go/internal/validator/folder_spec.go

```diff
@@ -61,16 +59,17 @@ func (v *validator) Validate() ve.ValidationErrors {
 	case "", "ga": // do nothing
 	case "beta":
 		if v.pkg.Version.Major() > 0 && v.pkg.Version.Prerelease() == "" {
-			errs = append(errs, errors.Errorf("spec for [%s] defines beta features which can't be enabled for packages with a stable semantic version", v.pkg.Path(v.folderPath)))
+			errs = append(errs, fmt.Errorf("spec for [%s] defines beta features which can't be enabled for packages with a stable semantic version", v.pkg.Path(v.folderPath)))
 		} else {
+			message := fmt.Sprintf("Warning: package with non-stable semantic version and active beta features (enabled in [%s]) can't be released as stable version.", v.pkg.Path(v.folderPath))
```

Comment:
There are some usages of this "Warning:" as an error.

Should we change when used as errors to `warning:` ?